.github/workflows/tests.yaml

```
@@ -30,8 +36,12 @@ jobs:
         run: |
           npm ci
           npm run build:lerna
-      - run: npm ci
-      - run: npm run test:server
+
+      - name: Install clean npm instance
```

It would use the same as below

```suggestion
      - name: Install npm dependencies
```